### ISSUE TITLE
License violation

### ISSUE THREAD

**Issue by chyyran** [Starz0r/winfsp-rs#1]
Hi,

As per https://github.com/SnowflakePowered/snowflake-projfs/blob/master/README.md, this code is licensed under the GPL v3 license so as to be compatible with the WinFSP license here: https://github.com/winfsp/winfsp/blob/master/License.txt

I kind of licensed it very lazily in the Readme because as you've probably notice the bindings are not ready for production yet and are somewhat tied to the semantics of snowflake-projfs. Regardless, please license your fork appropriately.

**Comment by Starz0r** [comment]
I was actually wondering what the license was if not GPL v3, since you didn't put any GPL-compatible one in the original repository (other than listing it in the readme, which I must have missed). The next time I get around to making changes in the repository, I'll make sure to properly include the entire license as a file.

**Comment by chyyran** [comment]
Yeah it was my bad not making it very explicit. I plan to spin the WinFSP parts off into its own repository after verifying its correctness and properly cleaning it up as its own crate.
